test/unit/ldp/http/SimpleTargetExtractor.test.ts

```diff
@@ -12,6 +12,11 @@ describe('A SimpleTargetExtractor', (): void => {
   });
 
   it('returns the input URL.', async(): Promise<void> => {
-    await expect(extractor.handle({ url: 'url' } as any)).resolves.toEqual({ path: 'url' });
+    await expect(extractor.handle({ url: 'url', headers: { host: 'test.com' }} as any)).resolves.toEqual({ path: 'http://test.com/url' });
```

Comment:
weird that there is no initial slash

Reply:
both work, I can change it if it bothers you ;)

Reply:
Not as much bothers as that `url` is invalid input; so it should probably reject?

Reply:
This string gets directly fed to the Node.js library [here](https://github.com/solid/community-server/pull/21/files#diff-ced3a2e7f39f8d267d767aba6818247dR22), so unless we only want to reject this specific instance that would be a bit more difficult. In practice this should never occur due to the URL parsing happening beforehand though

Reply:
OK.